pkgs/by-name/pl/plasmusic-toolbar/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation rec {
```

Comment:
Also, using `finalAttrs:` is preferred nowadays (when possible), because it makes overriding much easier.